api/docs/http_api_index.md

```diff
@@ -82,10 +98,6 @@ use standard HTTP response code to signify those errors.
 * When the server is unable to find an endpoint handler for the request that
   starts with `/v2.0`, it will return a 404 Not Found response
 
```

Comment:
Should this clause be removed or tweaked? The reference to "/v2.0" doesn't make sense any more.